lms/services/youtube.py

```diff
@@ -58,11 +58,19 @@ def video_info(self, video_id: str) -> dict:
             raise VideoNotFound(video_id) from err
 
         snippet = item["snippet"]
+        content_details = item["contentDetails"]
+        is_restricted = (
+            False
+            if "ytRating" not in content_details["contentRating"]
+            else content_details["contentRating"]["ytRating"] == "ytAgeRestricted"
+        )
```

Comment:
I found this info in stackoverflow, but the comment mentions it's explained in the official docs. I'm trying to find a link to add here as a comment.

Reply:
Done https://developers.google.com/youtube/v3/docs/videos#contentDetails.contentRating.ytRating